pkg/reconciler/route/resources/ingress_test.go

```diff
@@ -77,6 +81,7 @@ func TestMakeIngressCorrectMetadata(t *testing.T) {
 			// Make sure to get passdownIngressClass instead of ingressClass
 			networking.IngressClassAnnotationKey: passdownIngressClass,
 			"test-annotation":                    "bar",
+			traffic.RolloutAnnotationKey:         emptyRollout,
```

Comment:
Is there a test for non-empty case?

Reply:
not in this file. They're all captured in `table_test.go`